packages/security/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@loopback/security",
+  "version": "0.0.1",
+  "description": "A LoopBack component for security support.",
+  "engines": {
+    "node": ">=8.9"
+  },
+  "scripts": {
+    "acceptance": "lb-mocha \"dist/__tests__/acceptance/**/*.js\"",
+    "build": "lb-tsc",
+    "clean": "lb-clean loopback-authorization*.tgz dist package *.tsbuildinfo",
+    "integration": "lb-mocha \"dist/__tests__/integration/**/*.js\"",
+    "prepublishOnly": "npm run build && npm run build:apidocs",
+    "pretest": "npm run build",
+    "test": "lb-mocha \"dist/__tests__/**/*.js\"",
+    "unit": "lb-mocha \"dist/__tests__/unit/**/*.js\"",
+    "verify": "npm pack && tar xf loopback-security*.tgz && tree package && npm run clean"
+  },
+  "author": "IBM",
+  "copyright.owner": "IBM Corp.",
+  "license": "MIT",
+  "dependencies": {
+    "@loopback/context": "^1.13.0",
+    "@loopback/core": "^1.8.5",
+    "debug": "^4.1.1"
+  },
+  "devDependencies": {
+    "@loopback/build": "^1.5.2",
+    "@loopback/testlab": "^1.2.7",
+    "@types/debug": "^4.1.4",
+    "@types/node": "10.14.15",
+    "casbin": "^2.0.4"
+  },
+  "keywords": [
+    "LoopBack",
+    "Security"
+  ],
+  "files": [
+    "README.md",
+    "index.js",
+    "index.d.ts",
+    "dist",
+    "src",
+    "!*/__tests__"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/strongloop/loopback-next.git"
+  }
+}
```

Comment:
Let's compare with an existing module such as `@loopback/context` to make sure the package.json is consistent.